scripts/dasharo-deploy

```diff
@@ -734,9 +1024,13 @@ update() {
     print_ok "Successfully updated Dasharo firmware."
   fi
 
+  echo "Sending logs..."
```

Comment:
Maybe add this echo to `send_dts_logs` function to not confuse user as it only sends logs if it was enabled.

Reply:
Fixed [here](https://github.com/Dasharo/dts-scripts/compare/ed6c6089caa0e541f15150b15a79a7e26c6ae365..9aa00a04166f712f638acc875e4004c28ece5c32)